<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Employee</class>
 <widget class="QMainWindow" name="Employee">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Employee Window Style */
QMainWindow {
    background-color: #f5f5f5; /* Светло-серый фон для главного окна */
    border: 1px solid #dcdcdc; /* Граница окна */
}

/* Table Widgets Style */
QTabWidget {
    border: 2px solid #dcdcdc; /* Граница для таблицы */
    border-radius: 5px; /* Скругленные углы */
    background-color: #ffffff; /* Белый фон */
    font-size: 14px; /* Размер шрифта */
}

/* Table Header Style */
QHeaderView::section {
    background-color: #0078d7; /* Синий фон заголовка таблицы */
    color: #ffffff; /* Белый цвет текста */
    padding: 8px; /* Отступ внутри заголовка */
}

/* Combo Box Style */
QComboBox {
    border: 2px solid #dcdcdc; /* Граница для комбо-бокса */
    border-radius: 5px; /* Скругленные углы */
    padding: 5px; /* Отступ внутри комбо-бокса */
    font-size: 14px; /* Размер шрифта */
    background-color: #ffffff; /* Белый фон */
}

/* Line Edits */
QLineEdit {
    border: 2px solid #dcdcdc; /* Граница для полей ввода */
    border-radius: 5px; /* Скругленные углы */
    padding: 5px; /* Отступ внутри поля ввода */
    font-size: 14px; /* Размер шрифта */
    background-color: #ffffff; /* Белый фон */
}

/* Buttons */
QPushButton {
    background-color: #0078d7; /* Синий фон кнопки */
    color: #ffffff; /* Белый цвет текста */
    border: none; /* Убираем границу */
    border-radius: 5px; /* Скругленные углы */
    padding: 10px; /* Отступ внутри кнопки */
    font-size: 14px; /* Размер шрифта */
}

QPushButton:hover {
    background-color: #005bb5; /* Темно-синий фон при наведении */
}

QPushButton:pressed {
    background-color: #004494; /* Еще более темный фон при нажатии */
}

/* Labels */
QLabel {
    font-size: 14px; /* Размер шрифта */
    color: #333333; /* Темно-серый цвет текста */
}

/* Message Boxes */
QMessageBox {
    background-color: #ffffff; /* Белый фон для окон сообщений */
    border: 1px solid #dcdcdc; /* Граница окна сообщения */
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Текущие дополнительные работы</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableWidget" name="tableWidgetCurrent">
          <column>
           <property name="text">
            <string>Наименование</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Трудоёмкость</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Сложность</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Оплата</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Комментарий</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Комментарий представителя</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Подтвердить выполненную работу</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Введите наименование работы:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="CurrentWorksComboBox">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Комментарий:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="CommentLine">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QPushButton" name="ConfirmButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Подтвердить выполненную работу</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Список завершенных работ</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTableWidget" name="tableWidgetConfirmed">
          <column>
           <property name="text">
            <string>Наименование</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Трудоёмкость</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Сложность</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Оплата</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Комментарий представителя</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="CloseButton">
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Выйти из системы</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>730</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
